src/libraries/Microsoft.PowerFx.Core/IR/Nodes/BinaryOpKind.cs

```diff
@@ -17,11 +17,16 @@ internal enum BinaryOpKind
         AddDateAndTime, // Date + Time
         AddDateAndDay, // Date + Number (Days)
         AddDateTimeAndDay,
-        AddTimeAndMilliseconds, // Time + Number (Milliseconds)
+        AddTimeAndNumber, // Time + Number (typically fraction of a day)
```

Comment:
>AddTimeAndNumber [](http://example.com/codeflow?start=8&length=16)

Since this is an internal enumeration I believe it's ok to rename it, but will double check